Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1091,32 +1181,41 @@ private protected void UpdateMaterialAssetByDefault(MaterialData src, ModelImpor
             smr.sharedMesh = rec.mesh;
 
             // update bones
-            if (dflags.hasBones) {
+            if (dflags.hasBones)
```

Comment:
We probably should start discussing about the convention, first about the brackets.
Perhaps we can talk about this a bit in our weekly sync meeting ?

That aside, if possible, the PR to change the convention should be separated from the PR to add functionality

Reply:
Not sure why this was reformatted in the first place but I have reverted it back to the original convention. I agree lets discuss and lets keep it in a separate PR